frontend/src/utils/artifactSort.js

```diff
@@ -0,0 +1,43 @@
+import { sortMostRecent } from './sort';
+
+export function sortByDateEdited(a, b) {
+  return sortMostRecent(a, b);
+}
+
+export function sortByDateCreated(a, b) {
+  if (a.createdAt > b.createdAt || (a.createdAt && !b.createdAt)) {
+    return -1;
+  } else if (a.createdAt < b.createdAt || (!a.createdAt && b.createdAt)) {
+    return 1;
+  }
+  return 0;
+}
+
+export function sortByName(a, b) {
+  return a.name.localeCompare(b.name);
+}
+
+export function sortByVersion(a, b) {
+  const apr_regex = /\b\d{1}\.\d{1}\.\d{1}\b/;
```

Comment:
Stylistic point, but would you mind changing this to `aprRegex` instead? Snake case isn't very typical in our code, so camel case would be preferable.

Reply:
You got it. I'm afraid of snakes myself.

Reply:
fixed